packages/wordcount/src/index.ts

```diff
@@ -132,3 +135,6 @@ export function count( text, type, userSettings ) {
 			return 0;
 	}
 }
+
+// Export types for external usage
+export type { Strategy, Settings, UserSettings } from './types';
```

Comment:
Let us use this to get rid of the `Undocumented declaration.` in the README file.
```suggestion
export * from './types';
```